eng/pipelines/corefx-base.yml

```diff
@@ -165,18 +162,18 @@ jobs:
                 helixQueues: $(_helixQueues)
                 msbuildScript: $(_msbuildCommand)
                 framework: $(_framework)
-                helixToken: $(HelixApiAccessToken)
                 ${{ if eq(parameters.isOfficialBuild, 'true') }}:
                   isExternal: false
                   waitForCompletion: false
                   officialBuildId: $(Build.BuildNumber)
+                  helixToken: $(HelixApiAccessToken)
 
                 ${{ if eq(parameters.isOfficialBuild, 'false') }}:
```

Comment:
I assume the if count for the whole block below? If yes why don#t you need a helixToken if the build is not an official one?